### PR TITLE
Issues 232/270/223/273

### LINKED ISSUE #223: Rename Wordsalad
We should rename Wordsalad to Anagram

### LINKED ISSUE #232: Checkstyle
` <module name="VisibilityModifier"/>
`

### LINKED ISSUE #270: Rename everything to gewia
Rename packages etc from e-edu to gewia

### LINKED ISSUE #273: Delete solutionTemplates
Delete Solution Templates because they are unnecessary now

### PR DESCRIPTION
**Pull request by PAException**
- Added Checkstyle (#232)
- Refactored E-Edu to Gewia (#270)
- Refactored Wordsalad to Anagram (#223) 
- Deleted Solution Templates (#273)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ or in detached mode:
 
 You want to contribute to this project? Great!
 
-[Drafts & Documentation](https://github.com/E-Edu/draft-documents)<br>
-[Git Workflow (German)](https://github.com/E-Edu/general/blob/master/guides/conventions.md)<br>
+[Drafts & Documentation](https://github.com/E-Edu/draft-documents) <br>
+[Git Workflow (German)](https://github.com/E-Edu/general/blob/master/guides/conventions.md) <br>
```
Comment (msentwickler) [inline README.md:22]: You should change this also and request the creation of it.

File: docker-compose.yml [hunk 2]
```diff
@@ -6,9 +6,9 @@ services:
     container_name: task_database
     restart: always
     environment:
-      MYSQL_USER: e-edu
-      MYSQL_PASSWORD: e-edu
-      MYSQL_DATABASE: e-edu
+      MYSQL_USER: gewia
```
Comment (msentwickler) [inline docker-compose.yml:9]: Does the server admin know about the changes?

**Review by msentwickler — CHANGES_REQUESTED** [review]

**Comment by msentwickler** [comment]
Please rebase and resolve the conflicts

**Comment by msentwickler** [comment]
And it would be nice if you move the repo to gewia and change the README also.

File: README.md (hunk 1 above)
Comment (PAException) [reply README.md:22]: I will change it if its moved

**Comment by PAException** [comment]
> And it would be nice if you move the repo to gewia and change the README also.

Yeah soon but what do you mean with readme

File: docker-compose.yml (hunk 2 above)
Comment (PAException) [reply docker-compose.yml:9]: Probably not I'll change it

**Comment by msentwickler** [comment]
This one: https://github.com/E-Edu/task/blob/master/README.md

**Review by msentwickler — APPROVED** [review]

**Pull request merged**